modules/KalturaSupport/resources/mw.KWidgetSupport.js

```diff
@@ -858,7 +858,11 @@ mw.KWidgetSupport.prototype = {
 					var flavors = playmanifest.flavors;
 					if ( flavors && flavors.length === 1 ) {
 						srcToPlay = flavors[0].url;
-						deferred.resolve( srcToPlay );
+						if(flavors.length === 1 && flavors[0].bitrate ){
```

Comment:
You already checked for length===1 two lines above so this is redundant

Reply:
Also, the bitrate here is in kbps IIRC so normalize it already to bps as it comes like this in the manifests